lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -676,6 +726,147 @@
             ]
          }
       },
+      {
+         "type": "Microsoft.Insights/autoscaleSettings",
+         "apiVersion": "2015-04-01",
+         "name": "[variables('dbvmssName')]",
+         "location": "[resourceGroup().location]",
```

Comment:
Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

Reply:
The location of the autoscalesettings is in the ResourceGroup. ResourceGroup is already prompted for and I use ResourceGroup to obtain the location.